app/views/sites/show.html.erb

```diff
@@ -42,6 +40,22 @@
             <dd class="mb-0 pb-2 ps-3"><%= @site.public_send(attribute_name) %></dd>
           <% end %>
         </dl>
+
+        <dl class="show-page_dl d-grid row-gap-2 pt-3 border-top mb-0">
+          <% %i[delivery_address delivery_times].each do |attribute_name| %>
+            <dt class="pb-2"><%= Site.human_attribute_name(attribute_name) %></dt>
+            <dd class="mb-0 pb-2 ps-3"><%= @site.public_send(attribute_name) %></dd>
+          <% end %>
+
+          <dt class="pb-2"><%= Site.human_attribute_name(:delivery_map) %></dt>
+          <dd class="mb-0 pb-2 ps-3">
+            <% if @site.delivery_map.attached? %>
+              <%= link_to @site.delivery_map, rel: :noopener, target: :_blank do %>
+                <%= image_tag @site.delivery_map, class: "w-100" %>
```

Comment:
Use a representation with resize. Here if use upload a pdf, it will do an image of first page at least.